ui/src/projects.vue

```diff
@@ -180,6 +196,12 @@ export default {
                     } else {
                         this.other_projects.push(p);
                     }
+                    if(localStorage.getItem("recent_projectid")){
+                        const recentProjects = JSON.parse(localStorage.getItem("recent_projectid")).slice(0).slice(-5);
+                        if(recentProjects && recentProjects.includes(p._id.toString())) {
+                            this.recentProjects.push(p);
```

Comment:
Won't the list keep growing? Also.. if I visit a group, then  visit other groups (at least 5 of them), then the first group will never show on the recent projects list again, as that group is already listed in the array to the left?

Reply:
Yes, the list would keep growing and it's listed to the left!

Reply:
Will you be able to fix it?

Reply:
the project will show if it is accessed again. If not then only the last 5 will show up. I will still make always have no more than 5 projects.